README.md

```diff
@@ -588,6 +588,11 @@ JSON formatter for SimpleCov
 
 A formatter that generates a coverage badge for use in your project's readme using ImageMagick.
 
+### [simplecov-cobertura](https://github.com/dashingrocket/simplecov-cobertura)
+*by Jesse Bowes*
```

Comment:
Please add another `#`. i.e. `#### [simplecov-cobertura](https://github.com/dashingrocket/simplecov-cobertura)
*by Jesse Bowes*`

and amend commit